travis_test.m

```diff
@@ -5,9 +5,10 @@
 [s, v] = bfCheckJavaPath()
 bfInitLogging('INFO');
 
-test_bfopen('tubhiswt-3D/tubhiswt_C0.ome.tif');
+arg_list = argv();
+test_bfopen(arg_list{1});
 
 % Increase the debugging verbosity
 javaMethod('setRootLevel', 'loci.common.DebugTools', 'DEBUG');
 
-test_bfopen('tubhiswt-3D/tubhiswt_C0.ome.tif');
+test_bfopen(arg_list{2});
```

Comment:
Makes complete sense, the only reason for using 2 separate arguments was because of the length of the verbose logging for the multi-plane OME-TIFF. I can pick a smaller file and probably test both OME-TIFF and fake.